doc/themes/scikit-learn/static/css/sphinx_copybutton.css

```diff
@@ -0,0 +1,103 @@
+/*
```

Comment:
Why can't the CSS be kept under sphinxext/sphinx_copybutton as it would be if installed as a separate package?

Reply:
Sure. All the files are now under `sphinxext/sphinx_copybutton`.